htdocs/api/v0.0.3-dev/candidates/visits/Dicoms.php

```diff
@@ -0,0 +1,148 @@
+<?php
+/**
+ * Handles API requests for the candidate's dicom files
+ *
+ * PHP Version 5.5+
+ *
+ * @category Main
+ * @package  API
+ * @author   Mouna Safi-Harab <mouna.safiharab@gmail.com>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://www.github.com/aces/Loris/
+ */
+namespace Loris\API\Candidates\Candidate\Visit;
+require_once '../Visit.php';
+
+/**
+ * Handles API requests for the candidate's dicom files. Extends
+ * Visit so that the constructor will validate the candidate
+ * and visit_label portion of URL automatically.
```

Comment:
I got it from `Images.php` originally written by Dave, so I assume since he used URL, it is properly the right thing to do here (as the only real anglophone among the 3 of us)?

Reply:
no, @cmadjar is right. It's a typo. It's not grammatically correct without "the".

Reply:
ok then, will add `the` :)